client/state/google-my-business/reducer.js

```diff
@@ -4,12 +4,25 @@
  * Internal dependencies
  */
 import { combineReducers, createReducer, keyedReducer } from 'state/utils';
-import { GOOGLE_MY_BUSINESS_CONNECT_LOCATION } from 'state/action-types';
+import {
```

Comment:
Are those reducers maybe more appropriate for "ui state" `/state/ui/` ?

Reply:
Good question. My understanding is that `state/ui` should be used for any state that could be stored locally in the components themselves. The way we manage Google My Business data has changed with https://github.com/Automattic/wp-calypso/pull/23806 though, and relies more on site preferences now.

Reply:
Oh, I was tricked by the diff. I agree that we could move  `GOOGLE_MY_BUSINESS_STATS_CHANGE_INTERVAL` to `/state/ui` indeed.